resources/dicts/patrols/plains/hunting/greenleaf.json

```diff
@@ -114,7 +114,7 @@
         "chance_of_success": 40,
         "success_outcomes": [
             {
-                "text": "Your  cats are smaller, but they outnumber the fox, if only by a little, and it's a prize they're willing to fight for. With a prayer to StarClan they throw themselves into battle, and the fox, already half full and unwilling to lose blood over it, is driven off from the kill.",
+                "text": "Your cats are smaller, but they outnumber the fox, if only by a little, and it's a prize they're willing to fight for. With a prayer to StarClan they throw themselves into battle, and the fox, already half full and unwilling to lose blood over it, is driven off from the kill.",
```

Comment:
probably a good time to change "Your cats" to just "The patrol" while we're at it

Reply:
```suggestion
                "text": "The patrol are smaller, but they outnumber the fox, if only by a little, and it's a prize they're willing to fight for. With a prayer to StarClan, they throw themselves into battle. The fox, already half-full and unwilling to lose blood over it, is driven off from the kill.",
```